adapters/huaweiads/huaweiads.go

```diff
@@ -788,39 +778,60 @@ func convertCountryCode(country string) (out string) {
 	return defaultCountryName
 }
 
+func getCountryCodeFromMCC(MCC string) (out string) {
+	var countryMCC = strings.Split(MCC, "-")[0]
+	intVar, err := strconv.Atoi(countryMCC)
+
+	if err != nil {
+		return defaultCountryName
```

Comment:
Is there any way to trigger/cover this line of code

Reply:
banner4_mccmnc.json file is used to cover line 783.

Reply:
I still see this line uncovered when I run the tests in the huaweiads package